### DISCUSSION TITLE
Tag lines to appear in review

### DISCUSSION THREAD

**Discussion by Nathanpmyoung** [st3v3nmw/obsidian-spaced-repetition#287] [Q&A]
I've read around a bit, but can't find it, sorry if this is dumb.

I want to know the steps to tag individual lines so that they appear in review, bringing them back to my mind.

My ideal workflow would be
1. Tag a line with #sr or ::
2. It appears in review

I think this ought to work currently but no cards are appearing. How do I get the Spaced Repetition to search all my notes an pull out any that are tagged with some tag for review?

Thanks for building this.

**Comment by st3v3nmw** [comment]
Option 1: Add the `#sr` tag to settings under `Flashcard tags` (Have you?), then create the `::` card. This will only search notes with that `#sr` tag for flashcards.
Option 2: Turn on the `Convert folders to decks and subdecks?` option, then create the `::` card. This will search _all_ notes.
